### DISCUSSION TITLE
3060 V1 is running at 50MH on ETH

### DISCUSSION THREAD

**Discussion by Clubsport640** [Lolliedieb/lolMiner-releases#747] [General]
Just checking on my rigs and to my surprise, one of my Version 1 3060's is running at 49-50MH after it reboot itself an hour ago. Any thoughts on why and how to make it happen to the rest. Driver version is 460.39 and the fan has locked itself an 100% on that card also.
![Capture](https://user-images.githubusercontent.com/91775983/135707292-3fb4bb7c-8e7d-4623-8e5d-95d470f42f84.PNG)
..

**Comment by Ricco59163** [comment]
interresting ;)

**Comment by FantDev** [comment]
It's a known workaround in hiveos.  Get your 3060v1 to crash due to fan glitch and it then hashes fullrate, doesn't report fan speed, and fan runs at full 100%.

**Reply by LuiGGyFM** [reply]
Fan doesn't run always at 100%, that only happens if you forced stop the fan and then the glitch happens.

**Reply by FantDev** [reply]
That's what I said.  If you want the 3060 v1 to hash at the full hashrate.
